test_scripts/API/VehicleData/WindowStatus/014_App_version_is_more_than_version_windowStatus_in_DB_and_API.lua

```diff
@@ -0,0 +1,48 @@
+---------------------------------------------------------------------------------------------------
```

Comment:
Please update file name to` test_scripts/API/VehicleData/WindowStatus/014_App_version_is_greater_than_version_windowStatus_in_DB_and_API.lua`